galaxy_importer/main.py

```diff
@@ -16,37 +16,62 @@
 # along with Galaxy.  If not, see <http://www.apache.org/licenses/>.
 
 import argparse
+import json
 import logging
+from pprint import pprint
 import sys
 
 from . import collection
 
 
-def start_import(filepath):
-    """Outputs result of galaxy_importer import process.
+def main(args=None):
+    logging.basicConfig(
+        stream=sys.stdout,
+        format='%(levelname)s: %(message)s',
+        level=logging.INFO)
+    args = parse_args(args)
 
-    :param filepath: Artifact file to import.
-    """
-    json_data = collection.import_collection(filepath)
-    # TODO: make default output go to file
-    print(json_data)
+    json_data = call_importer(filepath=args.filepath)
+
+    if args.print_result:
+        pprint(json.loads(json_data))
```

Comment:
Is this for debug purposes or supposed to be used by parent program?

In any case using `pprint` for JSON formatting is a terrible idea. It is not guaranteed that `pprint` will provide a valid formatted JSON output. You should use `json.dump() / json.dumps()` with `indent=` parameter for prettifying JSON.

Reply:
This is just for debugging, updated.

@cutwater do you think we should remove the `json.dumps()` from what the galaxy_importer returns and let the caller do that?: https://github.com/ansible/galaxy-importer/blob/master/galaxy_importer/collection.py#L47

Reply:
@awcrosby yes, I think it's a better option, then you can decide on formatting when writing data to the output.